containers/concent-api/install-concent-api-python-dependencies.sh.j2

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
-python="/opt/pyenv/versions/{{ python_version }}.{{ python_patch_version }}/bin/python{{ python_version }}"
+DEFAULT_PYTHON_PATH="/opt/pyenv/versions/{{ python_version }}.{{ python_patch_version }}/bin/python{{ python_version }}"
```

Comment:
I'm not sure we really need this default. In `Dockerfile` you're passing `python3.6` so the default is not used. Is there any situation in which we do use this default and it's actually useful?

Reply:
Yes, we use default in other docker images. The `verifier` image uses `python3.6` package from system source repository after its based on the Ubuntu distribution.